src/GUI/src/CommandHandler.cpp

```diff
@@ -153,7 +153,7 @@ namespace GUI {
             int orientation = 0;
             Entities::EntityOrientation enumOrientation = Entities::EntityOrientation::UP;
 
-            if (!(ss >> cmd >> id >> x >> y >> orientation) || orientation < 0 || orientation > 3) {
+            if (!(ss >> cmd >> id >> x >> y >> orientation) || orientation < 1 || orientation > 4) {
```

Comment:
dafuk is 1 and 4 ?

Reply:
```suggestion
            if (!(ss >> cmd >> id >> x >> y >> orientation) || orientation < UP || orientation > LEFT) {
```
on peut utiliser les enum comme des macro